output.tf

```diff
@@ -1 +1,3 @@
-# Insert output values here, if no outputs are defined delete this file
\ No newline at end of file
+output "cloudfront_domain_name" {
+  value = aws_cloudfront_distribution.cloudfront_distribution.domain_name
```

Comment:
add descriptions

also add outputs for the distribution ID and other values that are likely to get used from a cloudfront distribution